chandra_models/xija/pftank2t/pftank2t_spec.json

```diff
@@ -145,7 +145,17 @@
             1110
         ]
     },
-    "limits": {},
+    "limits": {
+        "pftank2t": {
+            "odb.caution.high": 120,
+            "odb.caution.high.matlab_identifier": "yellow_limit",
+            "odb.warning.high": 140,
+            "odb.warning.high.matlab_identifier": "red_limit",
+            "planning.warning.high": 110,
+            "planning.warning.high.matlab_identifier": "reference_limit",
+            "unit": "degF"
+        }
```

Comment:
the MATLAB tools characteristics are also storing `pftankip` (tank pressure limits).  I don't think the xija models use these values, but wasn't sure if we wanted to store all associated limits in one place by adding them here.

Reply:
I suppose we could make a `pftankip_spec.json` model here that it not actually a xija model but has the `limits` data structure. That's slightly odd but would make it possible to change tank pressure planning limits in the same way. Fine by me.

Reply:
If these values aren't really being used/validated in the SOT tools at the moment, I'd prefer that they stay in the MATLAB Characteristics, where any changes will get some validation from the existing regression testing.  If they are used/tested in the SOT release workflow then I am happy either way.

Reply:
Tank pressure limits are not in the SOT tools to my knowledge. So 👍 to keeping them in the characterisitics.